docker/test/stateless/run.sh

```diff
@@ -55,6 +55,12 @@ echo "ASAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environ
 echo "UBSAN_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
 echo "LLVM_SYMBOLIZER_PATH=/usr/lib/llvm-10/bin/llvm-symbolizer" >> /etc/environment
 
+# /etc/environment will not be read in current shell
```

Comment:
Will it be read somewhere? Maybe just `export` right away, w/o `/etc/environment` at all?

Reply:
Yes, I guess initially it was done for clickhouse-server via service (there is `su` inside which will reset the env)